database/postgresql/src/main/java/com/example/javaopencommerce/item/ItemDetailsMatcher.java

```diff
@@ -0,0 +1,27 @@
+package com.example.javaopencommerce.item;
+
+import static java.util.stream.Collectors.toUnmodifiableList;
+
+import java.util.ArrayList;
+import java.util.List;
+
+interface ItemDetailsMatcher {
+  static List<Item> convertToItemModelList(List<ItemEntity> items,
+      List<ItemDetailsEntity> itemDetails) {
+
+    List<Item> itemModels = new ArrayList<>();
+    for (ItemEntity item : items) {
```

Comment:
This could be refactored to stream. New issue can be created for that :)

Reply:
done